### ISSUE TITLE
Poison Wyvern Monarch works on Gold Dragons and Black Dragons

### ISSUE THREAD

**Issue by GrafDeVan** [vcmi/vcmi#4364]
Current develop 1.6.0.257fb8c70c6a7746ad567f62bc348ea0d08d49d7

**Comment by kambala-decapitator** [comment]
Iirc not a bug

**Comment by Povelitelh3** [comment]
> Iirc not a bug

Yes, it's a bug. In the original heroes it doesn't work by default. But for the mod it's not a problem.

**Comment by krs0** [comment]
"targetCondition" : {
			"noneOf" : {
				"bonus.NON_LIVING" : "absolute",
				"bonus.MECHANICAL" : "absolute",
				"bonus.SIEGE_WEAPON" : "absolute",
				"bonus.UNDEAD" : "absolute",
				"bonus.GARGOYLE" : "absolute"
			}
		}

So what should be added to the list? Only black and gold dragons? All dragons?

https://heroes.thelazy.net/index.php/Spell_immunity

**Comment by GrafDeVan** [comment]
<img width="989" height="732" alt="Image" src="https://github.com/user-attachments/assets/87ca38dd-7243-41cf-8a8f-642df65fe2fa" />

<img width="957" height="819" alt="Image" src="https://github.com/user-attachments/assets/6e76bb4f-bd4e-4c21-887e-1f9652cb9ded" />

<img width="1316" height="139" alt="Image" src="https://github.com/user-attachments/assets/3be8afc0-8f2d-4f06-b32a-322494f2a14e" />

@krs0  Sorry, I didn't remember it from memory, so I took it from Fizmig 

**briefly, automatic translation**

The poison only affects the living.

Poison is a magical ability, but it doesn't have a level, unlike, for example, Scorpicore's Paralysis or Basilisk's Petrification. However, in general, it gravitates toward level 1 magic. Dragon immunities and the Dragonfather's Might artifact protect against Poison. The Medal of Vulnerability allows you to poison dragons. The Sphere of Forbidding, Cloak of Abjuration, or Cape of Silence do not protect against Poison. Poison works in Cursed Ground and in the Anti-Magic Garrison.
Poison cannot be removed by Dispel Spell. Anti-Magic does not protect against Poison and also does not remove it. The Heal spell can remove Poison and restore the health it reduces. Healing from the First Aid Tent does not remove Poison.
